File: Home Work/1. linearSearch.js
function linearSearch(arr, size, target) {
    for(let i = 0; i < size; i++) {
        if (arr[i] == target) {
            return i;
        }
    }
    return -1;
}

let arr = [2, 4, 5, 6 , 3, 8];
let target = 7;
let size = arr.length;
let result = linearSearch(arr, size, target);
(result == -1) ? console.log(`Element ${target} is not present in array.`) : console.log(`Element is present at index ${result}.`);

File: Home Work/1. linearSearch.cpp
#include<iostream>
using namespace std;

int linrearSearch(int arr[], int size, int target){
    for(int i = 0; i < size; i++){
        if(arr[i] == target){
            return i;
        }
    }
    return -1;
}

int main(){
    int myarr[] = {2, 4, 5, 2, 8, 0, 1, -4};
    int size = sizeof(myarr) / sizeof(int);
    // cout << size << endl;
    // cout << (sizeof(myarr) / sizeof(myarr[0])) << endl;
    int result =  linrearSearch(myarr, size, 0);
    (result == -1) 
        ? cout << "Element is not present in array."
        : cout << "Element is present at index : " << result;
    return 0;
}